package.json

```diff
@@ -18,6 +18,7 @@
     "testBlockchain": "npm run build:dist && node --stack-size=1500 ./tests/tester -b --fork='Constantinople' --dist --excludeDir='GeneralStateTests'",
     "testBlockchainGeneralStateTests": "npm run build:dist && node --stack-size=1500 ./tests/tester -b --dist --dir='GeneralStateTests'",
     "testAPI": "tape ./tests/api/*.js",
+    "testManual": "tape ./tests/manual/*.js",
```

Comment:
I assume the standard tests cover sstore exhaustively; if that's the case, couldn't we drop `constantinopleSstoreTest.js` (and `manual` for now)? I know deleting code is hard, but I feel like it has served its purpose :)

Reply:
Have deleted and pushed.